include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -194,10 +194,10 @@ SEQAN3_CONCEPT aligned_sequence =
     requires { typename detail::unaligned_seq_t<t>; } &&
     requires (t v, detail::unaligned_seq_t<t> unaligned)
```

Comment:
I knowledge about concepts is rather superficial, I don not understand what is happening in the main body of his concept. Is this some kind of testing procedure for insert and erase gaps?

Reply:
I explain it to you in person